angular/projects/researchdatabox/form/src/app/component/repeatable.component.ts

```diff
@@ -108,15 +111,16 @@ export class RepeatableComponent extends FormFieldBaseComponent<Array<unknown>>
     await this.createElement(elemEntry);
   }
 
+  protected getNextLocalUniqueId(): string {
```

Comment:
`getNextLocalUniqueId` implies that the ids are related, which is no longer true.
Suggest `getLocalUniqueId` instead.